qa/L0_trace/test.sh

```diff
@@ -777,6 +777,7 @@ SERVER_ARGS="--allow-sagemaker=true --model-control-mode=explicit \
                 --load-model=simple --load-model=ensemble_add_sub_int32_int32_int32 \
                 --load-model=repeat_int32 \
                 --load-model=input_all_required \
+                --load-model=dynamic_batch \
```

Comment:
Were we missing this model before? Is it correctly added?

Reply:
Yes this is a breakage cause by one of my changes in L0_trace.
Tested the pipeline for this change fixes currently failing L0_trace